docs/drivers/azure.md

```diff
@@ -92,6 +93,7 @@ Environment variables and default values:
 | `--azure-subnet-prefix`         | `AZURE_SUBNET_PREFIX`         | `192.168.0.0/16`   |
 | `--azure-availability-set`      | `AZURE_AVAILABILITY_SET`      | `docker-machine`   |
 | `--azure-storage-type`          | `AZURE_STORAGE_TYPE`          | `Standard_LRS`     |
+| `--azure-custom-data`           | `AZURE_CUSTOM_DATA_FILE`      | -                  |
```

Comment:
I'm not sure how popular this flag would be, can we move this all the way to the end?